src/main/resources/db/migration/V1_0_6__update.sql

```diff
@@ -0,0 +1,29 @@
+-- adding tables for multi users access
+
+CREATE TABLE `webusers` (

```

Comment:
suggestion: `web_users` since we use underscore as delimiter.

Reply:
opinion: i prefer to use singular names when naming tables... because obviously every table would have an `s` (or whatever) at the end to denote plurality since every table would have multiple entries. if every table name is plural, this notion loses its value/meaning and you could then stick to singular.

i know this is an opinionated convention and others might have different ones, but since the db tables of steve already use the singular names... i would suggest to stay with that just for the sake of consistency, even if you dont agree with my approach.